code-reviews/recipes/Terraform.md

```diff
@@ -53,10 +58,16 @@ echo -e "\n\n>>> Terraform validate"
 terraform validate
 ```
 
-> TODO: Add example of how to use this in an AzDO build validation pipeline
-
-## Code Review Checklist
-
+### Code Review Checklist
 In addition to the [Code Review Checklist](../readme.md) you should also look for these terraform specific code review items
 
-> TODO: Add checklist
+- Verify Terraform project is configured to use Azure Storage as remote state backend
+- Verify remote state backend storage account key is stored in the key vault and not hard coded
```

Comment:
Should we say Key Vault as it's a specific technology? It is stored in a secure location and not hardcoded in a manner that it could be checked into the code repository... Something like that?

Reply:
Makes sense to me. Maybe we can mention Key Vault as an example? secure location (e.g. Azure  Key Valut).

Reply:
@ianphil  and @debugerr - updated based on the comment.